test/suites/examples/create-eks.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-eks-task
+  namespace: tekton
+spec:
+  description: |
+    Create an EKS cluster and MNG.
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin up.
+  - name: k8s-version
+    description: The K8s version used for the cluster.
+  - name: aws-region
+    description: The region where the cluster will be created.
+  steps:
+  - name: create-cluster
+    image: docker.io/weaveworks/eksctl:0.86.0
```

Comment:
Should this version of eksctl be parametized?

Reply:
This should be kept the same so we ensure it works properly. We can change this as desired if we want to update dependencies later.